src/db.py

```diff
@@ -29,8 +29,8 @@ def query(**kwargs):
       d = _search_gt(d, field, value)
     elif comp == "in":
       d = _search_in(d, field, value)
-    elif comp in ("not", "not eq"):
-      d = _search_not(d, field, value)
+    # elif comp in ("not", "not eq"):
+    #   d = _search_not(d, field, value)  # TODO: implement
```

Comment:
If you change it to `_search_not_eq` then it'll be implemented. Turns out I just named it incorrectly.